### PR TITLE
[FIX] Fix svr test

### PR DESCRIPTION
**Pull request by mathurinm**

### CODE REVIEW

File: sparse_ho/models/svr.py [hunk 1]
```diff
@@ -364,7 +374,10 @@ def get_jac_obj(self, Xs, ys, n_samples, sign_beta,
             ddual_var[n_samples:(2 * n_samples), 0]
 
         maskC = np.abs(alpha) == C
```
Comment (mathurinm) [inline sparse_ho/models/svr.py:376]: use isclose in this case too @Klopfe ?

**Review by agramfort — APPROVED** [review]
nice @mathurinm !

File: sparse_ho/models/svr.py (hunk 1 above)
Comment (agramfort) [reply sparse_ho/models/svr.py:376]: yes using == on floats is a bad idea

**Comment by QB3** [comment]
Just to keep track of what is happening: the problem is that some tests introduced in the SVR are slow.
The goal of the PR is to understand why and to fix this problem.

**Comment by mathurinm** [comment]
I merged the fixes for #101, the test is still slow

**Comment by mathurinm** [comment]
closing as we have started to address the issue in another sequence of PRs

**Pull request closed without merging**